ansible/roles/master/tasks/kubelet-configure.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Write the config files for kubelet
+  template: src=kubelet.j2 dest={{ kube_config_dir }}/kubelet
```

Comment:
``kubelet.kubeconfig`` would have different meaning. ``kubelet`` is configurable for ``kubelet`` component, ``kubectl.kubeconfig`` is configuration for connecting to ``apiserver``.